src/client/pmi2.c

```diff
@@ -440,7 +442,8 @@ PMIX_EXPORT int PMI2_Info_GetNodeAttr(const char name[],
     PMIX_INFO_LOAD(&info[0], PMIX_OPTIONAL, &val_optinal, PMIX_BOOL);
 
     *found = 0;
-    rc = PMIx_Get(&myproc, name, info, 1, &val);
+    /* TODO: does PMI2's "name" makes sense to PMIx? */
```

Comment:
I'm still trying to understand this. I'll create the issue tomorrow with the analysis.